classes/plugin/api.py

```diff
@@ -0,0 +1,4 @@
+import classes.network.Connection as con
```

Comment:
I am not to overly keen on that line.
I think it is the best to avoid shortening it, especially since it is only used once in this module